.gitignore

```
@@ -1 +1,2 @@
 *known_hosts*
+/homebrew/Brewfile.lock.json
```

You are missing an empty line at the end

```suggestion
/homebrew/Brewfile.lock.json

```